diff --git a/serialize/serialize.h b/serialize/serialize.h
index 147c605..97712d3 100644
--- a/serialize/serialize.h
+++ b/serialize/serialize.h
@@ -1332,7 +1332,7 @@ namespace serialize
 
     template <typename Stream> bool serialize_float_internal( Stream & stream, float & value )
     {
-        uint32_t int_value;
+        uint32_t int_value = 0;
         if ( Stream::IsWriting )
         {
             memcpy( &int_value, &value, 4 );
